lib/linebuffer.js

```diff
@@ -24,6 +24,31 @@ var dccount = 0;
 var httperr;
 var queue;
 
+// check response to see need for buffering into queue
+const checkResponse = (error, response, body) => {
```

Comment:
I would name this something like `needsRetry` - that it's returning a bool, and what that value is meant to indicate is not clear from the name.

Reply:
Good naming - liked - done!